apps/geo/mutations.py

```diff
@@ -0,0 +1,96 @@
+import graphene
+
+from django.utils.translation import gettext
+
+from geo.models import AdminLevel, Region
+from geo.schema import AdminLevelType, RegionDetailType
+from geo.serializers import AdminLevelGqlSerializer, RegionGqSerializer
+
+from project.models import Project
+from utils.graphene.error_types import CustomErrorType
+from utils.graphene.mutation import GrapheneMutation, PsDeleteMutation, generate_input_type_for_serializer
+
+from deep.permissions import ProjectPermissions as PP
+
+RegionInputType = generate_input_type_for_serializer(
+    'RegionInputType',
+    serializer_class=RegionGqSerializer,
+)
+
+AdminLevelInputType = generate_input_type_for_serializer(
+    'AdminLevelInputType',
+    serializer_class=AdminLevelGqlSerializer,
+)
+
+
+class CreateRegion(GrapheneMutation):
+    class Arguments:
+        data = RegionInputType(required=True)
+    model = Region
+    serializer_class = RegionGqSerializer
+    result = graphene.Field(RegionDetailType)
+
+    @classmethod
+    def check_permissions(cls, info, **_):
+        return True  # global permission is always true
+
+
+class CreateAdminLevel(GrapheneMutation):
+    class Arguments:
+        data = AdminLevelInputType(required=True)
+    model = AdminLevel
+    serializer_class = AdminLevelGqlSerializer
+    result = graphene.Field(AdminLevelType)
+
+    @classmethod
+    def check_permissions(cls, info, **_):
+        return True  # global permission is always true
+
+
+class RemoveProjectRegion(PsDeleteMutation):
+    class Arguments:
+        id = graphene.ID(required=True)
+        project_id = graphene.ID(required=True)
+    errors = graphene.List(graphene.NonNull(CustomErrorType))
+    result = graphene.Field(RegionDetailType)
+    permissions = [PP.permission.UPDATE_PROJECT]
+
```

Comment:
We already have this https://github.com/the-deep/server/blob/61da4fa2306c1707bffa2c88c78deef4d584ae53/apps/project/mutation.py#L290-L321
Can you confirm the requirements?